.buildkite/scripts/steps/functional/apm_cypress.sh

```diff
@@ -30,6 +30,6 @@ echo "--- APM Cypress Tests"
 
 cd "$XPACK_DIR"
 
-node plugins/apm/scripts/test/e2e.js \
+node plugins/observability_solution/apm/scripts/test/e2e.js \
```

Comment:
Tests failed with the following error when applying the `ci:all-cypress-suites` label.

```
node:internal/modules/cjs/loader:1147
  throw err;
  ^
Error: Cannot find module '/var/lib/buildkite-agent/builds/kb-n2-4-spot-861f752882a4b7f7/elastic/kibana-pull-request/kibana/x-pack/plugins/apm/scripts/test/e2e.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

Reply:
Will be also fixed in this: https://github.com/elastic/kibana/pull/177949 - so if your PR won't be merged before that one, you can rely on the update from main, and won't need Operations' our approval

Reply:
Thanks!